ci-tooling/lib/projects/factory/gitlab.rb

```diff
@@ -41,33 +41,43 @@ def self.understand?(type)
 
     private
 
-    # FIXME: same as in Neon except component is merged
     def split_entry(entry)
       parts = entry.split('/')
-      name = parts[-1]
-      component = parts[0..-2].join('_') || 'gitlab'
-      [name, component]
+      name = parts.pop
+      component = parts.pop
```

Comment:
Is it intentional that this doesn't have the gitlab fallback anymore?

Reply:
Yes, it makes no sense to have 'gitlab' fallback, as gitlab have at least 1 level. There is never going to be gitlab.com/repo it will be always gitlab.com/group/repo.

Reply:
Is that also true for self-hosted gitlabs though? I suspect Rohan had a reason to put that there.

Reply:
AFAIU, yes. I can't really find any gitlab instance without top level groups